sql/driver/conn.go

```diff
@@ -123,32 +109,23 @@ func (c *conn) send(args Request) (*rows, error) {
 		}
 	}
 	result := resp.Results[index]
-	r.columns = make([]string, len(result.Columns))
-	for i, column := range result.Columns {
-		r.columns[i] = column
+	resultRows.columns = make([]string, 0, len(result.Columns))
+	for _, column := range result.Columns {
+		resultRows.columns = append(resultRows.columns, column)
```

Comment:
Similar to the comment above, using `append` doesn't seem any clearer to me than using indexing. Is there any particular reason we're making a copy of `result.Columns`? What about:

```
  resultRows.columns = result.Columns
```